[2022-11-20T13:15:51.353+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Call_Center_ETL_DAG.insert_call_info_data manual__2022-11-20T13:15:12.651027+00:00 [queued]>
[2022-11-20T13:15:51.464+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Call_Center_ETL_DAG.insert_call_info_data manual__2022-11-20T13:15:12.651027+00:00 [queued]>
[2022-11-20T13:15:51.467+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T13:15:51.469+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-20T13:15:51.471+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T13:15:51.889+0000] {taskinstance.py:1383} INFO - Executing <Task(MySqlOperator): insert_call_info_data> on 2022-11-20 13:15:12.651027+00:00
[2022-11-20T13:15:51.938+0000] {standard_task_runner.py:55} INFO - Started process 913 to run task
[2022-11-20T13:15:51.986+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Call_Center_ETL_DAG', 'insert_call_info_data', 'manual__2022-11-20T13:15:12.651027+00:00', '--job-id', '172', '--raw', '--subdir', 'DAGS_FOLDER/call_center_dag.py', '--cfg-path', '/tmp/tmpbrp3l8qn']
[2022-11-20T13:15:52.008+0000] {standard_task_runner.py:83} INFO - Job 172: Subtask insert_call_info_data
[2022-11-20T13:15:52.516+0000] {task_command.py:376} INFO - Running <TaskInstance: Call_Center_ETL_DAG.insert_call_info_data manual__2022-11-20T13:15:12.651027+00:00 [running]> on host 9eba6bc17a61
[2022-11-20T13:15:53.153+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Abdelrhman Yassein
AIRFLOW_CTX_DAG_ID=Call_Center_ETL_DAG
AIRFLOW_CTX_TASK_ID=insert_call_info_data
AIRFLOW_CTX_EXECUTION_DATE=2022-11-20T13:15:12.651027+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-20T13:15:12.651027+00:00
[2022-11-20T13:15:53.155+0000] {mysql.py:84} INFO - Executing: 
        INSERT INTO call_info(call_id,reason,channel,call_center,response_time)
        SELECT DISTINCT
            id as call_id,
            reason,
            channel,
            call_center,
            response_time
        FROM 
            staging_call
        WHERE
            id IS NOT NULL;
            
[2022-11-20T13:15:53.268+0000] {base.py:71} INFO - Using connection ID 'mysql_conn_id' for task execution.
[2022-11-20T13:15:53.400+0000] {sql.py:315} INFO - Running statement: 
        INSERT INTO call_info(call_id,reason,channel,call_center,response_time)
        SELECT DISTINCT
            id as call_id,
            reason,
            channel,
            call_center,
            response_time
        FROM 
            staging_call
        WHERE
            id IS NOT NULL;
            , parameters: None
[2022-11-20T13:15:57.395+0000] {sql.py:324} INFO - Rows affected: 32941
[2022-11-20T13:15:57.556+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=Call_Center_ETL_DAG, task_id=insert_call_info_data, execution_date=20221120T131512, start_date=20221120T131551, end_date=20221120T131557
[2022-11-20T13:15:57.650+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-20T13:15:57.701+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
